server/index.ts

```diff
@@ -9,13 +9,22 @@ import MoviesRouter from './api/movies';
 import ActorsRouter from './api/actors';
 import DirectorsRouter from './api/directors';
 import GenresRouter from './api/genres';
+import passport from 'passport';
+import passportLocal from 'passport-local';
+import cookieParser from 'cookie-parser';
+import session from 'express-session';
+import formData from 'express-form-data';
+// const dotenv = require('dotenv');
+const app = express();
+import { Response, Request } from 'express';
```

Comment:
this should be `import type`

Reply:
Just updated this, now reading
`import type { Response, Request} from 'express;`